### PR TITLE
keep track of hidden items

### PR DESCRIPTION
**Pull request by kalikaneko**
This addresses bug described in #72

### CODE REVIEW

**Comment by kalikaneko** [comment]
this includes #70, I can rebase when that gets merged.

File: systray_windows.go [hunk 1]
```diff
@@ -567,6 +568,30 @@ func (t *winTray) showMenu() error {
 	return nil
 }
 
+func (t *winTray) delFromVisibleItems(val int32) {
+	for i, itemval := range t.visibleItems {
+		if uint32(val) == itemval {
+			newvisible := append(t.visibleItems[:i], t.visibleItems[i+1:]...)
+			t.visibleItems = newvisible
```
Comment (meskio) [inline systray_windows.go]: I guess the compiler will produce the same code, but AFAIK the go `style` is to assign it in the append:
```
t.visibleItems = append(t.visibleItems[:i], t.visibleItems[i+1:]...)
```

Comment (meskio) [reply systray_windows.go]: I guess you can break here.

**Comment by joesis** [comment]
LGTM. Thanks @kalikaneko!

**Pull request merged**